packages/ui/src/app/pages/Profile/components/AccountItem.tsx

```diff
@@ -163,12 +171,32 @@ export const AccountItem = ({ account }: AccountItemDataProps) => {
 
   const [isDropped, setDropped] = useToggle()
 
+  const displayLockIcons = () => {
+    if (!balance || !balance.locks.length) {
+      return null
+    }
+
+    return balance.locks.map((lock) => lockIcon(lock.id || 1))
+  }
+
+  const displayLocks = () => {
+    if (!balance || !balance.locks.length) {
+      return <TextMedium light>No locks found.</TextMedium>
+    }
+
+    return balance.locks.map((lock) => <LockItem lock={lock} />)
+  }
+
   return (
     <>
       <AccountItemWrap key={address}>
         <AccountInfo account={account} />
         <TokenValue value={balance?.total} />
-        <TokenValue value={balance?.locked} />
+        <div>
+          <TokenValue value={balance?.locked} />
+          <br />
+          {displayLockIcons()}
```

Comment:
Can you extract a component for this? I see that we might display account locks elsewhere as well so it will be helpful.

Also please create a storybook story for this new component that will display available locks.

Reply:
✅